packages/discovery-provider/integration_tests/tasks/user_bank_mock_transactions.py

```diff
@@ -51,7 +51,7 @@
 # Pubkey.find_program_address([bytes(USDC_MINT)], Pubkey.from_string(CLAIMABLE_TOKENS_PDA))
 USDC_PDA = "7vKR1WSmyHvBmCvKPZBiN66PHZqYQbXw51SZdwtVd9Dt"
 
-# contentType:contentId:blockNumber
+# contentType:contentId:blockNumber:purchaseUserId:accessType
```

Comment:
This comment feels slightly misleading as most of the memos below don't include accessType?